### DISCUSSION TITLE
Will u add 10250 response for nodes

### DISCUSSION THREAD

**Discussion by nkuacac** [kubernetes-sigs/kwok#159] [General]
Using fake-kubelet, apiserver cpu raise to 100%. There are many logs like:
```
1 status.go:71] apiserver received an error that is not an metav1.Status: &fmt.wrapError{msg:"error trying to reach service: dial tcp 192.168.5.5:10250: connect: connection refused",
```

**Comment by wzshiming** [comment]
Thank you

This 10250 is the port exposed by kubelet, which I think is very necessary for simulation, and this feature will be tried to add in the future.

**Comment by wzshiming** [comment]
I sent an issue #162 to track this enhancement

**Reply by nkuacac** [reply]
Also，need simulate cadvisor metrics for fake nodes。Tks

**Reply by wzshiming** [reply]
I'm not quite sure what the point of this is, but I've created an issue #199 to track this enhancement, but it should be a lower priority, or someone willing to work on it.

**Comment by wzshiming** [comment]
Hi @nkuacac 

Now that Kwok v0.1.0 has been released, it implements a Kubelet service stubs, which should prevent that apiserver errors

**Reply by nkuacac** [reply]
Get, I will try later. Here is some example for metrics.
![image](https://user-images.githubusercontent.com/283912/217700644-2d309ef5-2867-4682-a287-f7ef2c04c417.png)
